subprojects/packagefiles/sqlite3/meson.build

```diff
@@ -141,6 +143,9 @@ sqlite3_dep = declare_dependency(
   include_directories: depinc,
   link_with: libsqlite3,
 )
+if meson.version().version_compare('>=0.54.0')
```

Comment:
I'd update the minimum version instead.

Reply:
SQLite is widely used and pretty low in the dependency stack, so I figure it's worth a couple conditionals to avoid narrowing its Meson support.

Reply:
Oh indeed.  I do think there's some value in supporting distro-packaged Meson though.